scripts/build_clone_docs.sh

```diff
@@ -1,25 +1,60 @@
 # Clone the docs (draft or published)
 #
 # git clone does not like to clone into folders that are populated.  We are doing
-# this sequence of commands to workaround that limitation. 
+# this sequence of commands to workaround that limitation.
 # Could _not_ use:
 #   git clone git@github.com:kabanero-io/docs.git --branch develop src/main/content
 DOCS_GIT_URL=${1:-"https://github.com/kabanero-io/docs"}
 DOCS_GIT_REVISION=${2:-"master"}
 
 CUR_DIR="$(cd $(dirname $0) && pwd)"
+
+git_clone_doc_tags() {
+    #get all tags from the repo
+    TAGS=$(git ls-remote --tags $1 | cut -d/ -f3-)
+    #get the latest tag from repo
+    LATEST_TAG=$(git ls-remote --tags $DOCS_GIT_URL | awk -F'/' '/[0-9].[0-9].[0-9].*/ { print $3}' | sort -nr | head -n1)
+    
+    for TAG in $TAGS; do
+    #Remove the letter v or any other letter before the version number so VERSION only contains the numaric value of a release
+    VERSION=$(sed 's/[A-Za-z]*//g' <<<$TAG)
```

Comment:
all tags won't have a v in front anymore as we decided that's the way it should be. I fixed the v0.0.4 tag as well